tests/services/test_feed_validator.py

```
@@ -0,0 +1,172 @@
+@pytest.mark.asyncio
+async def test_ssl_tls_issue():
+    validator = FeedValidator()
+    url = "https://example.com/ssl-tls"
+    with patch("aiohttp.ClientSession.get", new_callable=AsyncMock) as mock_get:
+        mock_get.side_effect = aiohttp.ClientSSLError
+        is_valid, error = await validator.validate_feed(url)
+        assert not is_valid
+        assert error is not None
+
+@pytest.mark.asyncio
+async def test_successful_retry_after_failure():
```

Ruff (0.8.0)

Missing return type annotation for public function `test_successful_retry_after_failure`

Add return type annotation: `None`

(ANN201)